csrankings.csv

```diff
@@ -8726,6 +8727,7 @@ Meng Jiang 0001,University of Notre Dame,http://www.meng-jiang.com,LZIPfCkAAAAJ
 Mengchi Liu,Carleton University,http://www.scs.carleton.ca/~mengchi,NOSCHOLARPAGE
 Mengjie Zhang,Victoria University of Wellington,http://ecs.victoria.ac.nz/Main/MengjieZhang,hLvGrrkAAAAJ
 Merrick L. Furst,Georgia Institute of Technology,http://www.cc.gatech.edu/people/merrick-furst,NOSCHOLARPAGE
+Metin Sitti,Max Planck Institute,http://pi.is.mpg.de/person/sitti,YU4Ce_MAAAAJ

```

Comment:
This person is not on this list, which I believe is canonical:

https://wp.mpi-sws.org/cis-copy-mpi-sws-org/people/

Reply:
This true, but we'll add him soon. But as you can see, he's listed here:https://www.is.mpg.de/person/sitti

By the way, the page is just a temporary page, the original one is: https://www.cis.mpg.de

Thanks,
CArina

Reply:
Can you go ahead and add him and let me know when this is done? Thanks.

Reply:
We updated the page:  https://www.cis.mpg.de and Metin Sitti is showing up there.
Hopefully, you can now confirm the changes.

Thanks,
CArina

Reply:
Yep, thanks!